toc/rfc/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Foundry Request For Comments
+
+Cloud Foundry community members use this directory as a forum to make, discuss and record both process and technical decisions. The outcomes of these discussions can be either an Action Plan, a new Standard or Process that the Cloud Foundry community should follow. This repository is open as a reference for all working groups and for the wider community.
+
+## Process
+
+1. Create a new branch on this repo and copy `rfc-000-template.md` to `rfc-000-my-proposal.md` and edit.
+1. If your RFC is only relevant for a single Working Group, please place it in a directory for the working group.
```

Comment:
Is this directory supposed to be the designated directory for the working group under `toc/working-groups`, or an identically named directory under `toc/rfc`, or something else?

Reply:
I've added some clarifying text